model/simulationtests/swimmingpool_indoor.rb

```diff
@@ -0,0 +1,138 @@
+require 'openstudio'
+require 'lib/baseline_model'
+
+model = BaselineModel.new
+
+#make a 1 story, 100m X 50m, 5 zone core/perimeter building
+model.add_geometry({"length" => 100,
+              "width" => 50,
+              "num_floors" => 1,
+              "floor_to_floor_height" => 4,
+              "plenum_height" => 0,
+              "perimeter_zone_depth" => 3})
+
+#add windows at a 40% window-to-wall ratio
+model.add_windows({"wwr" => 0.4,
+                  "offset" => 1,
+                  "application_type" => "Above Floor"})
+
+#add thermostats
+model.add_thermostats({"heating_setpoint" => 19,
+                       "cooling_setpoint" => 26})
+
+#assign constructions from a local library to the walls/windows/etc. in the model
+model.set_constructions()
+
+#set whole building space type; simplified 90.1-2004 Large Office Whole Building
+model.set_space_type()
+
+#add design days to the model (Chicago)
+model.add_design_days()
+
+# Add ASHRAE System type 07, VAV w/ Reheat, this creates a ChW, a HW loop and a
+# Condenser Loop
+model.add_hvac({"ashrae_sys_num" => '07'})
+
+boilers = model.getBoilerHotWaters.sort_by{|c| c.name.to_s}
+
+heating_loop = boilers.first.plantLoop.get
+
+core_zone = model.getThermalZoneByName("Story 1 Core Thermal Zone").get
+floor_surface = core_zone.spaces[0].surfaces.select{|s| s.surfaceType == 'Floor'}[0]
+
+# Create a SwimmingPoolIndoor. This mimics the 5ZoneSwimmingPool E+ example
+swimmingPoolIndoor = OpenStudio::Model::SwimmingPoolIndoor.new(model, floor_surface)
```

Comment:
Rest of test will test explicitly each setter to be a perfect demonstration of how to use this object